src/cobald/daemon/core/logger.py

```diff
@@ -21,7 +21,7 @@ def initialise_logging(level: str, target: str, short_format: bool):
         raise SystemExit(
             "invalid log level %r, expected any of 'DEBUG',"
             "'INFO', 'WARNING', 'ERROR' or 'CRITICAL'" % level
-        )
+        ) from None
```

Comment:
Should it not link to the "`attribute_error`" above?

Reply:
Since this is called to process "user" input, I don't think the entire trace is relevant. The purpose is to preserve for the user that they used an invalid log level; knowing that we tried fetching it from ``logging`` doesn't seem to add anything of use for them.

Reply:
Thanks a lot. Agreed